portal/import_template_ui.php

```diff
@@ -431,18 +464,69 @@ function checkCategory() {
                     }
                     if (empty($template_id)) {
-                        echo '<tr><td>' . $none_message . "</td></tr>\n";
+                        echo '<tr><td></td><td>' . $none_message . "</td></tr>\n";
+                    }
+                    echo "</tbody>\n";
+                    echo "</table>\n";
+                    ?>
+                </div>
+            </div>
+            <hr />
+            <!-- Send Profiles -->
+            <div class='row'>
+                <div class='col col-12'>
+                    <div class="h5">
+                        <i class='fa fa-eye mr-1' data-toggle='collapse' data-target='#profile_send_collapse' role='button' title="<?php echo xlt('Click to expand or collapse Send Profile panel.'); ?>"></i><?php echo xlt('Send Profiles') ?>
+                        <span class="mx-auto"><button class="btn btn-sm btn-primary mb-1" onclick="return popProfileDialog()"><?php echo xlt('Manage Profiles') ?></button></span>
+                    </div>
+                </div>
+                <!-- Send Profiles table -->
+                <div class='col col-12 table-responsive <?php echo attr($_REQUEST['profile_send_state'] ?: 'collapse') ?>' id="profile_send_collapse">
+                    <?php
+                    echo "<table class='table table-sm table-striped table-bordered'>\n";
+                    echo "<thead>\n";
+                    echo "<tr>\n" .
+                        "<th style='width: 5%'>" . xlt('Send') . '</th>' .
+                        '<th style="min-width: 25%">' . xlt('Profile') . '</th>' .
+                        '<th style="max-width: 65%">' . xlt('Assigned Templates') . '</th>' .
+                        '<th>' . xlt('Total') . '</th>' .
+                        "</tr>\n";
+                    echo "</thead>\n";
+                    foreach ($profile_list as $profile => $profiles) {
+                        $template_list = '';
+                        $profile_items_list = $templateService->getProfileListByProfile($profile);
+                        if (empty($profile_items_list)) {
+                            continue;
+                        }
+                        $total = 0;
+                        foreach ($profile_items_list as $key => $files) {
+                            $total += count($files ?? []);
+                            foreach ($files as $file) {
+                                $template_list .= $file['template_name'] . '; ';
+                            }
+                        }
+                        $profile_esc = attr($profile);
+                        echo '<tr>';
+                        echo "<td><input type='checkbox' class='form-check-inline' name='send' data-send_profile='yes' value='" . $profile_esc . "' /></td>";
+                        echo '<td>' . text($profiles['title']) . '</td>';
+                        echo '<td>' . text($template_list) . '</td>';
+                        echo '<td>' . attr($total) . '</td>';
```

Comment:
woohooo, one more very minor escaping issue, would use text() above

Reply:
I have no idea why my brain blocks escaping out! I know how to do this... Jiminy!:)
Thanks